toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -227,18 +275,24 @@ func (b *LiveOSIsoBuilder) stageIsoMakerInitrdArtifacts(writeableRootfsDir, isoM
 		return fmt.Errorf("failed to create %s\n%w", targetBootloadersDir, err)
 	}
 
-	sourceBoot64EfiPath := b.artifacts.bootx64EfiPath
-	targetBoot64EfiPath := filepath.Join(targetBootloadersDir, bootx64Binary)
-	err = file.Copy(sourceBoot64EfiPath, targetBoot64EfiPath)
+	arch, err := getArchitecture()
+	if err != nil {
+		return err
+	}
+
+	sourceBootEfiPath := b.artifacts.bootEfiPath
+	targetBootEfiPath := filepath.Join(targetBootloadersDir, bootloaderFilesConfig[arch].bootBinary)
+	err = file.Copy(sourceBootEfiPath, targetBootEfiPath)
 	if err != nil {
 		return fmt.Errorf("failed to stage bootloader file (bootx64.efi):\n%w", err)
 	}
 
-	sourceGrub64EfiPath := b.artifacts.grubx64EfiPath
-	targetGrub64EfiPath := filepath.Join(targetBootloadersDir, grubx64Binary)
-	err = file.Copy(sourceGrub64EfiPath, targetGrub64EfiPath)
+	sourceGrubEfiPath := b.artifacts.grubEfiPath
+	targetGrubEfiPath := filepath.Join(targetBootloadersDir, bootloaderFilesConfig[arch].grubBinary)
+
+	err = file.Copy(sourceGrubEfiPath, targetGrubEfiPath)
 	if err != nil {
-		return fmt.Errorf("failed to stage bootloader file (grubx64.efi):\n%w", err)
+		return fmt.Errorf("failed to stage bootloader file :\n%w", err)
```

Comment:
It might be nice to include `bootloaderFilesConfig[arch].grubBinary` in the error message.